Documentation/CHANGELOG.md

```diff
@@ -25,7 +25,7 @@ PR: [#832](https://github.com/econ-ark/HARK/pull/832). See [this forthcoming REM
 * Using Lognormal.from_mean_std in the forward simulation of the RiskyAsset model [#1019](https://github.com/econ-ark/HARK/pull/1019)
 * Fix bug in DCEGM's primary kink finder due to numpy no longer accepting NaN in integer arrays [#990](https://github.com/econ-ark/HARK/pull/990).
 * Add a general class for consumers who can save using a risky asset [#1012](https://github.com/econ-ark/HARK/pull/1012/).
-
+Add Boolean attribute 'PerfMITShk' to PerfForesightConsumerType. When true, allows perfect foresight MIT shocks to be simulated. [#1013](https://github.com/econ-ark/HARK/pull/1013).
```

Comment:
Novice users might not realize that adding this to PerfForesightConsumerType also adds it to most of our other consumption models, because most of them inherit from PerfForesightConsumerType.  Revise this to be vaguer:  "most consumption models" instead of "PerfForesightConsumerType"